Dockerfile

```
@@ -25,6 +25,21 @@ COPY Makefile Makefile
 COPY database/sqlite database/sqlite
 RUN make sqlite-db-refresh
 
+# INSTALL POSTGRESQL
+RUN wget https://ftp.postgresql.org/pub/source/v14.0/postgresql-14.0.tar.gz \
+    && tar -zxf postgresql-14.0.tar.gz \
+    && cd postgresql-14.0 \
+    && ./configure --without-readline --without-zlib\
+    && make \
+    && make install 
+
```

set the PATH variable so that psql can be accessed from inside the docker container
```suggestion
 export PATH="/usr/local/pgsql:${PATH}"
```